x-pack/docs/en/watcher/managing-watches.asciidoc

```diff
@@ -33,4 +33,4 @@ GET .watches/_search
   "size" : 100
 }
 --------------------------------------------------
-// TEST[setup:my_active_watch]
+// TEST[skip:deprecation warning]
```

Comment:
what is the plan for this documentation?

Reply:
This is listed in https://github.com/elastic/elasticsearch/issues/62501, to be updated by core/features. While it's not quite a test in the same vein as many of the others, it appears as though it may be an indication that there is a Watcher feature missing (similar to some of the other tests).